app/assets/stylesheets/blacklight/_bookmark.scss

```
@@ -18,3 +18,16 @@ label.toggle-bookmark {
 		display: none;
 	}
 }
+
+.toggle-bookmark .blacklight-icons svg {
+  height: 1.25rem;
+  width: 1.25rem;
+  path {
+    fill:none;
```

```suggestion
    fill: none;
```
Please add a space here.